build-tools/packages/build-tools/src/repoPolicyCheck/handlers/npmPackages.ts

```diff
@@ -580,7 +582,7 @@ export const handlers: Handler[] = [
 					}
 				}
 
-				writeFile(file, JSON.stringify(json, undefined, 2) + newline);
+				writeFile(file, JSON.stringify(json, undefined, indentation) + newline);
```

Comment:
```suggestion
				writeJsonSync(file, json, { spaces: "\t" });
```

Reply:
The point is to use the detected indentation, not force tabs.